src/lib/Behat/BrowserContext/UserPreferencesContext.php

```diff
@@ -8,15 +8,34 @@
 
 use Behat\Behat\Context\Context;
 use Ibexa\AdminUi\Behat\Page\ChangePasswordPage;
+use Ibexa\AdminUi\Behat\Page\UserSettingsPage;
 
 class UserPreferencesContext implements Context
 {
-    /** @var \Ibexa\AdminUi\Behat\Page\ChangePasswordPage */
-    private $changePasswordPage;
+    private ChangePasswordPage $changePasswordPage;
 
-    public function __construct(ChangePasswordPage $changePasswordPage)
+    private UserSettingsPage $userSettingsPage;
+
+    public function __construct(ChangePasswordPage $changePasswordPage, UserSettingsPage $userSettingsPage)
     {
         $this->changePasswordPage = $changePasswordPage;
+        $this->userSettingsPage = $userSettingsPage;
+    }
+
+    /**
+     * @Given I switch to :tabName tab
```

Comment:
Here imo step should be more precise, maybe something like: `I switch to :tabName tab in User Settings`

Reply:
`User Settings` is better than the `User Preferences` I came up with 😄 👍